src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StatusRoutes.scala

```diff
@@ -17,15 +20,23 @@ object BuildTimeVersion {
 
 trait StatusRoutes {
   val statusService: StatusService
+  private val openTelemetryTags: Map[String, String] = Map("endpoint" -> "status")
   implicit val executionContext: ExecutionContext
+  implicit val openTelemetry: OpenTelemetryMetrics[IO]
 
   def statusRoutes: server.Route =
     pathPrefix("status") {
       pathEndOrSingleSlash {
         get {
           complete {
             statusService.getStatus().map { statusResponse =>
-              val httpStatus = if (statusResponse.ok) StatusCodes.OK else StatusCodes.InternalServerError
+              val httpStatus = if (statusResponse.ok) {
+                openTelemetry.incrementCounter("checkStatus-success", tags = openTelemetryTags).unsafeToFuture()
+                StatusCodes.OK
+              } else {
+                openTelemetry.incrementCounter("checkStatus-failure", tags = openTelemetryTags).unsafeToFuture()
+                StatusCodes.InternalServerError
+              }
```

Comment:
this is where the magic happens

This is using the global IORuntime, but I'm not entirely sure if it's ok to use that here.